### DISCUSSION TITLE
Will Jetstream support external DB like postgres for persistence

### DISCUSSION THREAD

**Discussion by raricent** [nats-io/nats-server#3507] [Q&A] [answered]
We used postgres with nats-streaming for persistence. We replicated the DB ourselves. With Jetstream, I do not see an option for postgres. Will it not be supported?

**Comment by derekcollison** [comment, marked as answer]
No, we will support memory and file based for the store level. We can replicate in either store and each store can also have digital twins or source mux/demux streams.

**Reply by raricent** [reply]
Thanks for the reply. Much appreciated.
